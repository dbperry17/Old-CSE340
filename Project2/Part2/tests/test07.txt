abcdefg 123456789012345678901234567890

222222222222222222222222222222222233333333333333333333444444444444444444444455555555556666666666777777777777777778888888888899999999

12333333333444444444455555555555566666666777777777777

