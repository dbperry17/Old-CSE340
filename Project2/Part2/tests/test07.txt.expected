5 NUM 12333333333444444444455555555555566666666777777777777
3 NUM 222222222222222222222222222222222233333333333333333333444444444444444444444455555555556666666666777777777777777778888888888899999999
1 NUM 123456789012345678901234567890
